### PR TITLE
Add device specs for GPS data collection

### LINKED ISSUE #702: Add GPS sensor information to device specifications
https://docs.communityhealthtoolkit.org/core/releases/#client-devices

### PR DESCRIPTION
**Pull request by eljhkrr**
Fixes #702

### CODE REVIEW

**Review by smbuthia — COMMENTED** [review]
Thanks for the addition @eljhkrr and sorry for taking so long to get to this PR.

A couple of questions I have:

- Is this a hard requirement for all GPS data collection? If not, then we should consider moving this section to a different doc because of the implication of the cost of acquiring such devices for CHVs
- Is A-GPS the same as [Assisted GPS](https://en.wikipedia.org/wiki/Assisted_GNSS). It would be great to have abbreviations in full at first use and with links to further reading where possible, especially for less common concepts.

**Comment by derickl** [comment]
We can add abbreviations to our glossary page https://docs.communityhealthtoolkit.org/glossary/

**Comment by eljhkrr** [comment]
@smbuthia 

- Expanded first use of A-GPS
- Deployment needs and quality of A-GPS service will dictate whether stand-alone GPS is necessary. Which section would be a better fit for this documentation? Added it to make it easier to find when considering device specs.

@derickl Added to glossary

File: content/en/core/releases/_index.md [hunk 1]
```diff
@@ -67,6 +67,15 @@ The following is the minimum specification recommendation for smartphones to han
 | RAM | 1GB |
 | Storage | 8GB |
 
+If CHWs will be collecting GPS data, the devices should include autonomous GPS sensors in addition to assisted GPS (A-GPS) for areas where GSM network connectivity isn't very good. Autonomous GPS is usually labeled in terms of the supported navigation satellite system:
```
Comment (smbuthia) [inline content/en/core/releases/_index.md:70]:
```suggestion
If CHWs will be collecting GPS data, autonomous GPS sensors in addition to assisted GPS (A-GPS) for areas with poor GSM network connectivity will enhance the quality of GPS data collected. Autonomous GPS is usually labeled in terms of the supported navigation satellite system:
```
I have rephrased this a bit.

**Review by smbuthia — APPROVED** [review]
One minor suggestion in line, otherwise looks good.

**Pull request merged**